notebooks/TradingCustomerChurnClassifierSparkML.jupyter-py36.ipynb

```diff
@@ -99,8 +101,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install --user pyspark==2.3.2\n",
-    "!pip install --user py4j==0.10.7"
+    "# The following libraries are required for this notebook and should already be installed in the \n",
+    "# the kernel. If not, install them here. The versions listed have been tested to work with this notebook.\n",
+    "\n",
+    "# NOTE: if you do have to perform one or more of these installs, you MUST restart the kernel\n",
+    "\n",
+    "#!pip install --user pyspark==2.3.2\n",
+    "#!pip install --user py4j==0.10.7\n",
+    "#!pip install --user watson-machine-learning-client==1.0.375\n",
+    "\n",
+    "# check if libraries are loaded\n",
+    "!pip freeze"
```

Comment:
??? You don't really want to see all that pip freeze do you?  I mean, I do the same thing but don't leave it in there.

Reply:
commented out